www/js/clickHandling.js

```diff
@@ -110,12 +110,31 @@ const sendMessageForImagePlaceholderWithTarget = innerPlaceholderSpan => {
   })
 }
 
+/**
+ * Use "X", "Y", "Width" and "Height" keys so we can use CGRectMakeWithDictionaryRepresentation in
+ * native land to convert to CGRect.
+ * @param  {!ReferenceItem} referenceItem
+ * @return {void}
+ */
+const reformatReferenceItemRectToBridgeToCGRect = referenceItem => {
+  referenceItem.rect = {
```

Comment:
This was needed so the pagelib didn't have to know about iOS specific CGRect bridging details. The pagelib keeps the rect as a standard JS ClientRect.